jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Context.java

```diff
@@ -38,7 +38,9 @@
 public interface Context extends Attributes, Decorator, Executor
 {
     /**
-     * @return the encoded context path of this Context
+     * @return the encoded context path of this {@code Context} or
+     *         {@code null} if the context does not have a path
+     *         (e.g. the {@link org.eclipse.jetty.server.Server.ServerContext}).
```

Comment:
I'd still remove the reference to `ServerContext` because it does not help.
The next question would be: "but _when_ do I **not** have a `ServerContext` so I get something useful?".
Instead, add "@see ContextHandler" and there you explain that it populates `getContextPath()`.